swapi/src/schema/schema.js

```diff
@@ -90,17 +117,17 @@ function rootConnection(name, swapiType) {
 var rootType = new GraphQLObjectType({
   name: 'Root',
   fields: () => ({
-    allFilms: rootConnection('Films', 'films'),
+    filmConnection: rootConnection('Films', 'films'),
```

Comment:
I'm curious what @schrockn thinks about this case. To me, the "xConnection" naming scheme seems less clear here, as a connection implies a source node, and these are just paginateable lists from the query's root. I think `allX` is actually clearer in this case.